### PR TITLE
Updated CHANGELOG for 2.36.0

### PR DESCRIPTION
**Pull request by ygong**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+# Release 2.36.0 25-04-2022
+
+- Upgraded Envoy to 1.22.0 (#684)
+- Added option to enable strict request validation in transcoder (#685)
```
Comment (qiwzhang) [inline CHANGELOG.md:4]: remove line 4, as it is not exposed to ESPv2 users.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+- Fixed a test-only race condition by protecting critical sections (#683)
+- Updated flag --service help text (#681)
+- Updated outdated flag --version help text (#680)
```
Comment (qiwzhang) [inline CHANGELOG.md:7]: remove line 5, 6, 7 as they are not user visible

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+- Added response gzip,brotli compression (#675)
+- Reverted "Fix service manage timeout (#676)" (#678)
+- Fixed service_management_config integration test (#677)
+- Fixed service manage timeout (#676)
```
Comment (qiwzhang) [inline CHANGELOG.md:11]: remove line 9, 10, 11,  as they are test related.

Comment (ygong) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (ygong) [reply CHANGELOG.md:4]: Do you mean remove this line "Added option to enable strict request validation in transcoder"?

File: CHANGELOG.md (hunk 2 above)
Comment (ygong) [reply CHANGELOG.md:7]: With these 3 lines removed, it looks like the CHANGELOG only has the following remaining?

- Upgraded Envoy to 1.22.0 (#684)
- Added response gzip,brotli compression (#675)

File: CHANGELOG.md (hunk 1 above)
Comment (ygong) [reply CHANGELOG.md]: Removed

File: CHANGELOG.md (hunk 2 above)
Comment (ygong) [reply CHANGELOG.md]: Removed.

**Review by nareddyt — APPROVED** [review]

**Review by qiwzhang — APPROVED** [review]

**Pull request merged**